core/unit_test/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ endforeach()
 ##--------------------------------------------------------------------------##
 ## MPI tests.
 ##--------------------------------------------------------------------------##
-if(Cabana_ENABLE_MPI)
+if(${Cabana_ENABLE_MPI})
```

Comment:
That is not needed!

Reply:
You can just use `if(Cabana_ENABLE_MPI)`.

Reply:
I fixed this